server/apps/immich/src/modules/immich-jwt/immich-jwt.service.ts

```diff
@@ -33,4 +34,24 @@ export class ImmichJwtService {
       };
     }
   }
+
+  public extractJwtFromHeader(req: Request) {
+    if (
+      req.headers.authorization &&
+      (req.headers.authorization.split(' ')[0] === 'Bearer' || req.headers.authorization.split(' ')[0] === 'bearer')
```

Comment:
According to the [RFC](https://datatracker.ietf.org/doc/html/rfc6750#section-2.1) the lower case version is not allowed. I would be strict about that and not accept it.

Reply:
Ok, I will add in a fixed for the mobile app